### PR TITLE
upgrade: use the actual checks for the error modal

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by Itxaka** [comment]
[(3) UI: Generic error handling (landing page)](https://trello.com/c/oo5N6XNL/204-3-ui-generic-error-handling-landing-page)

File: assets/app/features/upgrade/controllers/upgrade-landing.controller.js [hunk 1]
```diff
@@ -160,13 +160,17 @@
                         vm.mode.type = response.data.best_method;
 
                         var checksValidity = [];
+                        var checksErrors = [];
```
Comment (santiph) [inline assets/app/features/upgrade/controllers/upgrade-landing.controller.js]: please reuse the previous `var` statement

**Comment by santiph** [comment]
no issues found. apporved

**Review by skazi0 — APPROVED** [review]
lgtm

**Pull request merged**